libs/ui/breadcrumb/helm/src/lib/breadcrumb-link.directive.ts

```diff
@@ -0,0 +1,17 @@
+import { computed, Directive, input } from '@angular/core';
+import { hlm } from '@spartan-ng/ui-core';
+
+@Directive({
+  selector: '[hlmBreadcrumbLink]',
+  standalone: true,
+  host: {
+    '[class]': '_computedClass()',
+  },
+})
+export class HlmBreadcrumbLinkDirective {
```

Comment:
I guess the breadcrumb will usually link to pages inside the angular application, thus `RouterLink` can be used.

You can add `RouterLink` as `hostDirectives` directive and expose the inputs. Usage for the link could look like

```html
<li hlmBreadcrumbItem>
    <a hlmBreadcrumbLink link="/home">Home</a>
</li>
```

`HlmPaginationLinkDirective` has a similar use case and uses `RouterLink` under the hood.

https://github.com/spartan-ng/spartan/blob/95292eee9ba446fa58b4f1e2590c8119df284c7e/libs/ui/pagination/helm/src/lib/hlm-pagination-link.directive.ts#L15-L45

Reply:
I update it base on your `hlmPaginationLink`. It's really nice

Reply:
Perfect!